hacs.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "multimatic",
+  "domains": ["vrc", "vaillant", "multimatic"]
```

Comment:
I think `domains` is the list of HA domains, so it should be: `["binary_sensor", "climate", "fan", "sensor", "water_heater"]`.